content/en/docs/instrumentation/net/libraries.md

```diff
@@ -26,46 +26,51 @@ dotnet add package OpenTelemetry.Instrumentation.{library-name-or-type}
 It is typically then registered at application startup time, such as when
 creating a [TracerProvider](/docs/concepts/signals/traces/#tracer-provider).
 
+## Note on Versioning
+
+The Semantic Conventions (Standards) for attribute names are not currently
+stable the instrumentation package is currently not in a released state. That
```

Comment:
```suggestion
The Semantic Conventions (Standards) for attribute names are not currently
stable and the instrumentation package is currently not in a released state. That
```

Reply:
missed "therefore", which makes it make sense